src/protocolgame.cpp

```diff
@@ -926,6 +927,14 @@ void ProtocolGame::parseHouseWindow(NetworkMessage& msg)
 	addGameTask(&Game::playerUpdateHouseWindow, player->getID(), doorId, id, text);
 }
 
+void ProtocolGame::parseWrapItem(NetworkMessage& msg)
+{
+	const Position& position = msg.getPosition();
+	uint16_t spriteId = msg.get<uint16_t>();
+	uint8_t stackPos = msg.getByte();
+	addGameTaskTimed(DISPATCHER_TASK_EXPIRATION, &Game::playerWrapItem, player->getID(), position, stackPos, spriteId);
```

Comment:
Any particular reason why this task is timed?

Reply:
Nope, this code is copied from the old wrap furniture pr, also every other "useItem" etc is timed, so thats why probably.